src/resources/postcss/ian-client.pcss

```
@@ -73,95 +71,107 @@
       align-items: center;
 
       a {
+        margin-right: 8px;
+
         font-family: var(--tec-font-family-sans-serif);
         font-size: var(--tec-font-size-2);
-        margin-right: 8px;
       }
     }
   }
 
   &__content {
+    position: relative;
+
     height: 100%;
-    background: var(--tec-admin-core-grey-0);
 
-    position: relative;
+    background: var(--tec-admin-core-grey-0);
   }
 
-  &__optin, &__empty {
-    text-align: center;
+  &__optin,
+  &__empty {
     padding: 40% 16px 16px;
 
+    text-align: center;
+
     &--title {
       font-family: var(--tec-font-family-sans-serif);
       font-size: var(--tec-font-size-4);
       font-weight: var(--tec-font-weight-bold);
     }
 
     &--description {
-      max-width: 320px;
       margin: 0 auto;
+      max-width: 320px;
 
       p {
+        margin-bottom: 10px;
+
         font-family: 'Arial', sans-serif;
         font-size: var(--tec-font-size-2);
-        margin-bottom: 10px;
       }
     }
 
     &--button {
+      margin: 16px auto;
       padding: 8px 10px;
```

```suggestion
      padding: var(--tec-spacer-1) var(--tec-spacer-2);
```
--tec-spacer-3 is 12px, we don't have a 10px spacer.